examples/crossplane-xr-multiversion/gitops/lib.sh

```
@@ -0,0 +1,1062 @@
+
+gitops_install_argo() {
+  echo "Installing Argo CD chart ${GITOPS_ARGO_CHART_VERSION}…"
+  helm repo add argo https://argoproj.github.io/argo-helm >/dev/null
+  helm repo update argo >/dev/null
+  helm upgrade --install argocd argo/argo-cd \
+    --namespace argocd --create-namespace \
+    --version "${GITOPS_ARGO_CHART_VERSION}" \
+    --set configs.cm."timeout\.reconciliation"=30s \
+    --wait --timeout 10m
+
+  kubectl -n argocd delete secret repo-${GITOPS_APP_NAME} --ignore-not-found
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)